java/org/apache/tomcat/websocket/Constants.java

```diff
@@ -114,6 +114,22 @@
     // Milliseconds so this is 20 seconds
     public static final long DEFAULT_BLOCKING_SEND_TIMEOUT = 20 * 1000;
 
+    // Configuration for idle read timeout on websocket channel
+    public static final String WS_READ_IDLE_TIMEOUT =
+            "org.apache.tomcat.websocket.WS_READ_IDLE_TIMEOUT";
+
+    // Configuration for idle write timeout on websocket channel
+    public static final String WS_WRITE_IDLE_TIMEOUT =
+            "org.apache.tomcat.websocket.WS_WRITE_IDLE_TIMEOUT";
+
+    // Configuration for idle write timeout on websocket channel
+    public static final String WS_CLOSE_READ_IDLE_TIMEOUT =
+            "org.apache.tomcat.websocket.WS_CLOSE_READ_IDLE_TIMEOUT";
+
+    // Configuration for idle write timeout on websocket channel
```

Comment:
Same copy/paste error here

Reply:
Removed the parameters which are not used